uaclient/config.py

```diff
@@ -254,6 +254,10 @@ def write_cache(self, key: str, content: "Any") -> None:
                 mode = 0o644
         util.write_file(filepath, content, mode=mode)
 
+    def save_file_with_suffix(self, file_name: str, suffix: str) -> None:
```

Comment:
I'm not sure we need this utility function on UAConfig. we can content = read_cache("machine-token");  write_cache("machine-token-saved", content)